src/Core/Domain/CustomerService/QueryHandler/GetCustomerServicesSummaryHandler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\CustomerService\QueryHandler;
+
+use Contact;
+use CustomerThread;
+use PrestaShop\PrestaShop\Core\Domain\CustomerService\QueryResult\CustomerServiceSummary;
+use Symfony\Component\Routing\RouterInterface;
+
+class GetCustomerServicesSummaryHandler implements GetCustomerServicesSummaryHandlerInterface
+{
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    public function __construct(RouterInterface $router)
+    {
+        $this->router = $router;
+    }
+
+    /**
+     * @return CustomerServiceSummary[]
+     */
+    public function handle(): array
```

Comment:
A query/command handler MUST be related to a query/command, even if this query doesn't have any parameters

You need to create a `GetCustomerServicesSummary` query bound to this handler, and you also should NEVER call the handler manually (like you do in the controller) The whole point of the CQRS architecture is that you don't know what is executed behind the scene, you just ask the bus to handle it and it's supposed to return you some data (or perform an action)